test/e2e/opm_test.go

```diff
@@ -489,6 +489,7 @@ var _ = Describe("opm", func() {
 	Context("using docker", func() {
 		cmd := exec.Command("docker")
 		cmd.Stderr = os.Stderr
+		cmd.Stdout = nil
```

Comment:
Doesn't look like we're reading `os.Stderr` for this command, so we might as well do both since an `error` returned by `cmd.Run()` will indicate a non-zero exit code regardless -- see https://pkg.go.dev/os/exec#Cmd.Run

P.S. `cmd.Stdout = nil` is the same as `cmd.Stdout = os.DevNull` -- see https://cs.opensource.google/go/go/+/refs/tags/go1.17.1:src/os/exec/exec.go;l=102-103

Reply:
In the case that cmd.Run returns an error, a non-zero exit code will only be logged when handling that error:

```bash
package main

func main() { ... }

func run() error {
	cmd := exec.Command("podman")
	cmd.Stderr = os.Stderr
	if err := cmd.Run(); err != nil {
		return err
	}
	return nil
}
```

```bash
$ go run main.go
Error: missing command 'podman COMMAND'
Try 'podman --help' for more information.
exit status 125
```

When removing that call to `cmd.Stderr = os.Stderr` the following is returned instead, which makes debugging local testing environments more difficult:

```bash
$ go run main.go
exit status 125
```

Weirdly enough, it looks like running the `docker` command logs everything to stderr locally, despite the return code being 0 - podman seems to handle logging errors to stderr correctly.